source/zh-cn/docs/index.md

```diff
@@ -18,7 +18,7 @@ Hexo 是一个快速、简洁且高效的博客框架。Hexo 使用 [Markdown](h
 
 安装 Hexo 相当简单。然而在安装前，您必须检查电脑中是否已安装下列应用程序：
 
-- [Node.js](http://nodejs.org/) (Should be at least nodejs 6.9)
+- [Node.js](http://nodejs.org/) (Should be at least Node.js 8.6)
```

Comment:
Maybe a quick translation in the meantime? :wink:

Reply:
zh-tw?

Reply:
@curbengh I am not sure the translations for zh-tw is correct, so I leave it untouched.

Reply:
I think should be the same (in this case). anyway, I added it.